wiki/Beatmap/Circle_size/de.md

```diff
@@ -0,0 +1,39 @@
+---
+tags:
+  - CS
+  - key count
+  - keycount
+---
+
+# Circle Size
+
+*Für Vorschriften zur Circle Size, siehe: [Rankingkriterien](/wiki/Ranking_criteria)*
+
+Die **Circle Size** (***CS***) ist eine Schwierigkeitseinstellung einer [Beatmap](/wiki/Beatmap), die die Größe von [Hit-Objekten](/wiki/Gameplay/Hit_object) beeinflusst. Die Circle Size kann Werte von 0 bis 10 annehmen, der [Beatmapeditor](/wiki/Client/Beatmap_editor) erlaubt aber nur Werte von 2 bis 7. Um andere Werte zu verwenden, muss die [`.osu`-Datei](/wiki/Client/File_formats/osu_(file_format)) der Beatmap manuell bearbeitet werden.
```

Comment:
Just for your information, I always wrote `Ranking-Kriterien` and `Beatmap-Editor`.

Reply:
Good catch. I should stick to that